library.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "printf",
+  "version": "0.6.3",
+  "description": "Standalone printf/sprintf formatted printing function library",
+  "keywords": "printf, sprintf, formatted printing",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eyalroz/printf.git"
+  },
+  "authors": [
+    {
+      "name": "Eyal Rozenberg",
+      "email": "eyalroz1@gmx.com",
+      "maintainer": true
+    }
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/eyalroz/printf",
+  "frameworks": "*",
+  "platforms": "*",
+  "export": {
+    "include": ["src"]
```

Comment:
Are you sure this won't export `printf.c` as though it was an include file?

Reply:
This option is for `pio pkg pack`, which creates the tarball for the library to be downloaded by the user if the library is published. In this case, only the `src` folder will be downloaded. This option is independent from the include files, which are automatically handled separately by the library dependency finder.

Reply:
Done